src/main/java/digital/slovensko/autogram/core/SigningParameters.java

```diff
@@ -234,6 +236,7 @@ private static SigningParameters buildParameters(SignatureLevel level, ASiCConta
                 packaging = eformAttributes.packaging();
                 xsdIdentifier = eformAttributes.xsdIdentifier();
                 xsltParams = eformAttributes.xsltParams();
+                embedUsedSchemas |= eformAttributes.embedUsedSchemas();
```

Comment:
Toto je zvlastny zapis, preco je tu `|=` ? Nemalo by tu byt `=` ako vsade inde + default?

Reply:
Pretože pri ORSR formulári môže autoLoad zistiť, že je to ORSR formulár a musí mu nastaviť embedded schémy. Avšak, ak je to aj obyčajný formulár a chceme embedovať schémy, tak to neprepíše naspäť na false. To je skrátený zápis `embedUsedSchemas = embedUsedSchemas || eformAttributes.embedUsedSchemas();`